[2021-04-21 16:28:23,112] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this 2021-04-19T00:00:00+00:00 [queued]>
[2021-04-21 16:28:23,120] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this 2021-04-19T00:00:00+00:00 [queued]>
[2021-04-21 16:28:23,120] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-04-21 16:28:23,120] {taskinstance.py:1069} INFO - Starting attempt 1 of 1
[2021-04-21 16:28:23,120] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-04-21 16:28:23,131] {taskinstance.py:1089} INFO - Executing <Task(BashOperator): also_run_this> on 2021-04-19T00:00:00+00:00
[2021-04-21 16:28:23,133] {standard_task_runner.py:52} INFO - Started process 68287 to run task
[2021-04-21 16:28:23,144] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'also_run_this', '2021-04-19T00:00:00+00:00', '--job-id', '5', '--pool', 'default_pool', '--raw', '--subdir', '/Users/wschrep/FreelanceWork/VIAA/IIIF_newproject/python_env/lib/python3.8/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/var/folders/68/c_pd8_vj55d9mdvv3mvllkh00000gn/T/tmpntvnms6m', '--error-file', '/var/folders/68/c_pd8_vj55d9mdvv3mvllkh00000gn/T/tmpn5tvbefv']
[2021-04-21 16:28:23,146] {standard_task_runner.py:77} INFO - Job 5: Subtask also_run_this
[2021-04-21 16:28:23,185] {logging_mixin.py:104} INFO - Running <TaskInstance: example_bash_operator.also_run_this 2021-04-19T00:00:00+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2021-04-21 16:28:23,296] {taskinstance.py:1281} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=also_run_this
AIRFLOW_CTX_EXECUTION_DATE=2021-04-19T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-04-19T00:00:00+00:00
[2021-04-21 16:28:23,299] {bash.py:135} INFO - Tmp dir root location: 
 /var/folders/68/c_pd8_vj55d9mdvv3mvllkh00000gn/T
[2021-04-21 16:28:23,301] {bash.py:158} INFO - Running command: echo "run_id=scheduled__2021-04-19T00:00:00+00:00 | dag_run=<DagRun example_bash_operator @ 2021-04-19 00:00:00+00:00: scheduled__2021-04-19T00:00:00+00:00, externally triggered: False>"
[2021-04-21 16:28:23,370] {bash.py:169} INFO - Output:
[2021-04-21 16:28:23,393] {bash.py:173} INFO - run_id=scheduled__2021-04-19T00:00:00+00:00 | dag_run=<DagRun example_bash_operator @ 2021-04-19 00:00:00+00:00: scheduled__2021-04-19T00:00:00+00:00, externally triggered: False>
[2021-04-21 16:28:23,395] {bash.py:177} INFO - Command exited with return code 0
[2021-04-21 16:28:23,476] {taskinstance.py:1185} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=also_run_this, execution_date=20210419T000000, start_date=20210421T142823, end_date=20210421T142823
[2021-04-21 16:28:23,521] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-04-21 16:28:23,546] {local_task_job.py:146} INFO - Task exited with return code 0
